indico/modules/events/editing/schemas.py

```diff
@@ -383,6 +385,30 @@ class ReviewCommentSchema(mm.Schema):
     internal = fields.Boolean(missing=False)
 
 
+class RoleSchema(mm.Schema):
+    name = fields.String()
+    code = fields.String()
+    source = fields.Method('_get_source')
+
+    def _get_source(self, role):
+        return 'category' if isinstance(role, CategoryRole) else 'event'
+
+
+class ServiceUserSchema(mm.ModelSchema):
+    class Meta:
+        model = User
+        fields = ('id', 'full_name', 'email', 'roles', 'event_manager')
+
+    roles = fields.Method('_get_roles')
+    event_manager = fields.Function(lambda u, ctx: ctx['event'].can_manage(u, 'editing_manager'))
```

Comment:
I'm including people who are "just" editing managers here, since the service is only editing-related so there should no need to differentiate between full event managers and editing managers.